spec/ruby/core/complex/polar_spec.rb

```diff
@@ -10,6 +10,22 @@
     ->{ Complex.polar(nil)      }.should raise_error(TypeError)
     ->{ Complex.polar(nil, nil) }.should raise_error(TypeError)
   end
+
+  ruby_version_is "3.2" do
```

Comment:
```suggestion
  ruby_bug "#19004", ""..."3.2" do
```

Reply:
Thanks for the review @nobu ! Just a quick clarification here: according to [the docs I saw](https://github.com/ruby/ruby/blob/master/spec/ruby/CONTRIBUTING.md#guard-for-bug) the use of `ruby_bug` means that the bug fix will be backported, but that is not something I need to do, correct?

Reply:
Yes, as already filled "Backport" field at that ticket, you don't have to do something there.